wordpress-mobile/AztecEditor-Android
# Disabled autocorrect on Samsung device with Samsung keyboard on API 33+

Fixes #1026

Autocorrect on Samsung have been broken for some time, which resulted in occasional text duplication.

This PR disables auto-correction on devices with Samsung with Android 13 that are using Samsung keyboard. We will still have suggestions and ability to manually commit them. I believe that this experience is better then what we have now.


### Test
1. Open the app using Samsung device with android 13 and Samsung keyboard.
2. Type some text into the app.
3. Confirm that nothing looks off - focus stays where it should be, image is present, styles are not being removed.
4. Confirm that autocorrect is not working - type something like "I hav " (with the space at the end) and notice that `hav` is not changed into `have`.
5. Switch to Gboard (or other keyboard).
6. Confirm that autocorrect is working - type something like "I hav " (with the space at the end) and notice that `hav` IS changed into `have`.
7. Change back to samsung keyboard and confirm that the autocorrect is still disabled.


Make sure strings will be translated:
